docs/extend/plugins_volume.md

```diff
@@ -230,3 +231,27 @@ Get the list of volumes registered with the plugin.
 ```
 
 Respond with a string error if an error occurred.
+
+### /VolumeDriver.Capabilities
+
+**Request**:
+```json
+{}
+```
+
+Get the list of volumes registered with the plugin.
+
+**Response**:
+```json
+{
+  "Capabilities": {
+    "Scope": "global"
```

Comment:
Will there be drivers that support both global *and* local? Don't know if that makes sense, but just wondering if this should be an array